### ISSUE TITLE
english voiceline in ion's mystic arte

### ISSUE THREAD

**Issue by silenthillfortheplaystation** [lifebottle/Tales-of-the-Abyss-Undub#7]
when using luke's mystic arte with the ion variant, ion's voiceline is in english.

**Comment by Kilicool64** [comment]
Are you sure you're using the latest version? That doesn't happen for me. I know this was an issue with the old undub before swosho came along to improve it.

**Comment by swosho** [comment]
Yeah, there's exactly zero English audio data left in the undubbed game since version 2.4 of the patch. English leftovers are simply impossible to encounter in the recent versions of the undub.

**Comment by Kilicool64** [comment]
Should be okay to close this, since it's demonstrably impossible for this user to have encountered the issue with a proper version of this patch.
